plugins/autowarmth.koplugin/main.lua

```diff
@@ -786,15 +786,15 @@ function AutoWarmth:getWarmthMenu()
             text_func = function()
                 if Device:hasNaturalLight() and self.control_warmth then
                     if self.warmth[num] <= 100 then
-                        return T(_("%1: %2 %"), text, self.warmth[num])
+                        return string.format("%s: %d %%", text, self.warmth[num])
```

Comment:
The localization might still opt for a different choice regarding a space before the percent sign or not though. Given that it's been there for a while I'd check what's been done.

Reply:
OK, thanks. There are some languages where there is done something with that string. So it is better to revert most of this.